GraphQL-TSC.md

```diff
@@ -169,19 +169,19 @@ Because we work in a distributed environment, the voting process must account fo
 * A notice is sent via email that the vote will conclude in three business days, reminding those who haven't voted that they should do so. The vote will conclude at the end of this time.
 * The election results will not change if additional TSC members vote.
 
-#### Ensuring efficient votes
+#### TSC voting member eligibility
 
-Should a TSC member miss three or more of the prior five meetings, they will no longer vote or be counted toward the total number of voting members. This is to ensure broad participation as well as to avoid voting deadlocks.
+A TSC member must attend two of the previous five meetings to be considered a "voting member" and be eligible to vote. Should a TSC member miss three or more of the prior five meetings, they will no longer vote or be counted toward the total number of voting members. This is to ensure broad participation as well as to avoid voting deadlocks.
 
-A voting member may regain their status by attending two consecutive meetings. They may participate in votes in the second meeting.
+A member may regain their voting status by attending two consecutive meetings. They may participate in votes in the second meeting.
 
-#### Resignation of voting members
+Elections of TSC members are exempted from this rule. All TSC members otherwise eligible to vote on a member election may do so regardless of their past attendance.
 
-If a rep resigns before the end of their term, the TSC may collect nominations and vote on a replacement representative to complete the term. The self-nomination period should be at least two weeks before holding a vote.
+#### Resignation or removal of a TSC member
 
-#### Removing a voting member
+A TSC member may voluntarily resign at any time before the end of their term by notifying the TSC chair. A TSC member may also be involuntarily removed by a supermajority (2/3) vote by eligible voting members less any with a conflict of interest.
 
-TSC members may be removed by a supermajority (2/3) vote by eligible voting members less any with a conflict of interest. The seat may then be filled for the remainder of the term.
+After a TSC seat vacancy the remaining TSC will collect nominations and vote on a replacement representative to complete the term. The self-nomination period should be at least two weeks before holding a vote.
```

Comment:
Unclear timing…should start, should end? There is likely to be a gap between nominations closing and a vote taking place. Perhaps “The self-nomination period should be at least two weeks, after which a vote shall be held.”?

Reply:
I like this, @benjie.